mobile/src/card-view.js

```diff
@@ -273,8 +273,16 @@ export class EditCardView extends Component {
       </View>
     )
   }
+  disableButton = () => {
+    if (this.state.firstName.length === 0 || this.state.lastName.length === 0 || !this.state.isChanged || this.state.mobile.match(/[a-z]/i)){
```

Comment:
Since you are returning a boolean, and the `if` condition is a boolean...
```js
disableButton = () => this.state.firstName.length === 0
  || this.state.lastName.length === 0
  || !this.state.isChanged
  || this.state.mobile.match(/[a-z]/i)
```

Maybe rename to `isButtonDisabled` since `disableButton` sounds like it takes an action to disable the button.